pkg/cmd/server/api/v1/conversions.go

```diff
@@ -19,6 +19,9 @@ func init() {
 			if obj.ServingInfo.RequestTimeoutSeconds == 0 {
 				obj.ServingInfo.RequestTimeoutSeconds = 60 * 60
 			}
+			if obj.ServingInfo.MaxRequestsInFlight == 0 {
+				obj.ServingInfo.MaxRequestsInFlight = 500
```

Comment:
I question the legitimacy of that tuning, given that nothing was being limited by this. Also, watches aren't supposed to fall under this